thecrew.css

```diff
@@ -17,485 +17,624 @@
 @import url(https://fonts.googleapis.com/css?family=Crimson+Text:400,700,900,400italic,700italic,900italic|Playfair+Display:400,700,900,400italic,700italic,900italic|Rock+Salt:400);
```

Comment:
this file was auto generated based on node sass:
`sass --no-source-map thecrew.scss thecrew.css`